src/RTG/ChatFilter/Loader.php

```diff
@@ -33,64 +33,41 @@
 use pocketmine\event\player\PlayerChatEvent;
 
 class Loader extends PluginBase implements Listener {
-
-    public $words;
+    public $cfg;
 
     public function onEnable() {
-        $this->getServer()->getPluginManager()->registerEvents($this, $this);
-        $this->words = array();
```

Comment:
I removed your $this->words field, yes. But in fact, you're defining the whole array again in line 44 (before this commit). This line is therefore unnecessary anyway.